weld/bin/repl.rs

```diff
@@ -25,8 +25,6 @@ use weld::sir::ast_to_sir;
 use weld::util::load_runtime_library;
 use weld::util::MERGER_BC;
 
-use weld::transforms;
```

Comment:
Needed to add this to avoid the following error I was seeing:

   Compiling weld v0.1.0 (file:///home/rpalamut/Projects/weld)
error[E0252]: a module named `transforms` has already been imported in this module
  --> weld/bin/repl.rs:28:5
   |
18 | use weld::*;
   |     -------- previous import of `transforms` here
...
28 | use weld::transforms;
   |     ^^^^^^^^^^^^^^^^ already imported

error[E0252]: a module named `transforms` has already been imported in this module
  --> weld/bin/repl.rs:28:5
   |
18 | use weld::*;
   |     -------- previous import of `transforms` here
...
28 | use weld::transforms;
   |     ^^^^^^^^^^^^^^^^ already imported

error: aborting due to previous error

error: aborting due to previous error

Build failed, waiting for other jobs to finish...
Build failed, waiting for other jobs to finish...
error: Could not compile `weld`.